project/pgp-sign.sbt

```diff
@@ -0,0 +1,6 @@
+credentials += Credentials(
+  "GnuPG Key ID",
+  "gpg",
+  "F638279A7281EE8EAE58C4B98D3CFE7E7D0262DA", // key identifier
```

Comment:
is this supposed to be public?

Reply:
Yeah, it's confusing and scary!

https://keyserver.ubuntu.com/pks/lookup?search=F638279A7281EE8EAE58C4B98D3CFE7E7D0262DA&fingerprint=on&op=index

It's the public key for lakeFS signing. We do the same in lakeFSFS.